course04/homework01/arne/main.cc

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <array>
+#include <algorithm>
+
+using namespace std;
+
+class sarray
+{
+public:
+    static constexpr const std::size_t MaxSize = 32;
```

Comment:
I checked this and indeed as Sergiu mentionned, constexpr and const are redundant, constexpr is enough.